### PR TITLE
Latency/Prefetch Testing

### PR DESCRIPTION
**Pull request by awatson1978**
Think I got the latency compensation working.  Works on localhost; lets see about on Travis.

### CODE REVIEW

**Comment by awatson1978** [comment]
This branch seems to be a dead end.  All of these tests are running 100% green on localhost.  So, I'm going to continue writing the multi-actor test script and get it running green; and we can then decide what to do about CI infrastructure.

**Comment by rbaertsch** [comment]
Is only a problem for collaboration and post or all of telescope ?

**Comment by awatson1978** [comment]
The `submit post` function is what's definitely not working on Travis.  How it's currently written, it's got a lot of dependencies that are required for it to work (subscriptions, cfs, avatar thumbnails, etc).   If any one of them isn't working or is slowed down, it can block the call.   That needs to be cleaned that up with async callbacks, futures, promises, after.insert hooks, and a few other things.  

But it's a little outside the scope of the current task of getting things under QA coverage.  If we want to go down that rabbit-hole and get this working on Travis, we can... but it will require a bit of re-architecture of the submit post function.

tl;dr - I _think_ this particular issue is mostly isolated to just the submit post functionality.  Looking at the app, I can tell that there are going to be prefetch issues throughout telescope because of the subscriptions within the templates.  But those aren't blocking right now.  What's blocking is this submit post functionality.
